config/locales/devise/en.yml

```diff
@@ -120,8 +120,7 @@ en:
         title: How should we send you a code?
         voice: Phone call
       otp_phone_label: Phone number
-      otp_phone_label_info: If you entered a landline, please select “Phone call”
-        below.
+      otp_phone_label_info: Mobile phone or landline
```

Comment:
I think we still want the previous help text because we've found that many people try to send SMS to a landline.

Reply:
The screenshots from the Jira issue just show `Mobile phone or landline` as the help text for the phone label input?